### PR TITLE
Add poco recipe

### PR DESCRIPTION
**Pull request by beenje**
I don't know much about windows, so I just skip it for now.
Not tested on OSX.

### CODE REVIEW

**Comment by beenje** [comment]
Hi @conda-forge/core,

This is a dependency for [Mantid](http://www.mantidproject.org/Main_Page) that I'd like added to conda-forge. Anyone has the time to give some feedback?
Thanks!

File: recipes/poco/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  skip: true    # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.0
+  host:
+    - openssl
+  run:
+    - openssl
```
Comment (hmaarrfk) [inline recipes/poco/meta.yaml]: No need for openssl in `run` https://github.com/conda-forge/openssl-feedstock/blob/master/recipe/meta.yaml

**Comment by hmaarrfk** [comment]
Maybe this can help 
https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=cmake#simple-cmake-based-bld-bat

**Comment by beenje** [comment]
Looks like this recipe has been added to conda-forge by someone else after this PR was opened: https://github.com/conda-forge/poco-feedstock

Can be closed then. Thanks for the comments!

**Pull request closed without merging**